src/main/java/CMV.java

```diff
@@ -118,7 +121,45 @@ private boolean LIC5() {
    * Computes the LIC 6 condition.
    */
   private boolean LIC6() {
-    // TODO: Implementation.
+
+    if (parameters.N_PTS < 3 || parameters.N_PTS > points.length) return false;
+    if (parameters.DIST < 0) return false;
```

Comment:
Unfortunately, according to the styleguide, we need to have braces and newlines even for one-line if-blocks: https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

I much prefer what you wrote, but I guess we should be consistent

Reply:
Thanks for pointing this out, will fix